x11-misc/compton/compton-9999.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
 EAPI=5
 
 PYTHON_COMPAT=( python3_{4,5} )
-inherit toolchain-funcs python-r1 git-2
+inherit toolchain-funcs python-r1 git-r3
 
 DESCRIPTION="A compositor for X, and a fork of xcompmgr-dana"
 HOMEPAGE="https://github.com/chjj/compton"
```

Comment:
also nix the `pkg_setup` and stick the `tc-export CC` into `src_compile`, as `src_compile` is only run when `${MERGE_TYPE} != binary`